docs/chapters/02_getting-started.md

```diff
@@ -373,11 +373,10 @@ After choosing your provider, you will see your project generated!:
 ℹ Project generated!
 ```
 
-> [!TIP] If you prefer to create the project with default parameters, you can run the command as `boost new:project booster-blog --default`. The default
+> [!TIP] If you prefer to create the project with default parameters, you can run the command as `boost new:project booster-blog --providerPackageName <package-name>`. The package name of the provider is the only required parameter in project creatio. The default
```

Comment:
```suggestion
> [!TIP] If you prefer to create the project with default parameters, you can run the command `boost new:project booster-blog --providerPackageName <package-name>`. The package name of the provider is the only required parameter and can be any `npm` package that implements Booster's provider interfaces. The framework provides the following: 

* `@boostercloud/framework-provider-aws`: Runs your app in AWS Lambda and stores the events, entity snapshots, and read models in DynamoDB.
* `@boostercloud/framework-provider-azure`: Runs your app in Azure Functions and stores the events, entity snapshots, and read models in CosmosDB.
* `@boostercloud/framework-provider-kubernetes` (Experimental): This package is a proof of concept that runs a Booster application on a Kubernetes cluster and Redis as the database.

The default
```

Reply:
I think that the DynamoDB, Cosmos, Lambda, etc... is too low-level for the getting started section. I'd remove that, specially with the K8s provider, given it is deprecated now 😅